infra/helper.py

```diff
@@ -240,7 +240,8 @@ def _get_work_dir(project_name=''):
 def _add_engine_args(parser):
   """Add common engine args."""
   parser.add_argument('--engine', default='libfuzzer',
-                      choices=['libfuzzer', 'afl', 'honggfuzz', 'none'])
+                      choices=['libfuzzer', 'afl', 'honggfuzz', 'perffuzz',
```

Comment:
Can we document somewhere (preferably the help message) that this is fuzzer is an academic experiment that we are playing around with and that this unsupported.

Reply:
I wonder where we can add this ? in infra/helper.py build_fuzzers and run_fuzzer or you were thinking docs ? putting in docs i feel makes more confusing ?

Reply:
Maybe just a comment somewhere then?